kuri_camera/src/ros_publisher.cpp

```diff
@@ -8,20 +8,17 @@
 #include <cv_bridge/cv_bridge.h>
 #include <mutex>
 #include <condition_variable>
-
-using std::string;
-using std::vector;
-
+#include <sensor_msgs/CompressedImage.h>
 
 struct KuriCameraROSPublisher {
 
-  image_transport::ImageTransport transport;
-  image_transport::CameraPublisher publisher;
+  ros::Publisher imagePublisher;
```

Comment:
ROS [conventionally uses snake case](http://wiki.ros.org/CppStyleGuide) for variables.

Reply:
fixed -- thanks!